docs/references/cli.rst

```diff
@@ -0,0 +1,41 @@
+.. _cli:
+
+CLI
+===
+
+PostgREST provides a CLI to start and run your postgrest service. The CLI provides the commands listed below:
+
+.. _cli_commands:
+
+CLI Commands
+------------
+
+Help and Version
+~~~~~~~~~~~~~~~~
+
+.. code:: bash
+
+  $ postgrest [-h|--help]
+  $ postgrest [-v|--version]
+
+
+Example Config
+~~~~~~~~~~~~~~
+
+.. code:: bash
+
+  $ postgrest [--example-file]
+  $ postgrest [--example-db]
+  $ postgrest [--example-env]
```

Comment:
@taimoorzaeem Btw, if you'd like you could split this and do another PR with just the docs commit. That one is already mergeable.

Of course you'd need to remove the `--example-db/--example/env` in that commit since they belong to this PR.

Reply:
Hmm, you're right. That would be much more manageable. Closing this PR.